gnomad_qc/v4/variant_qc/random_forest.py

```diff
@@ -104,13 +99,30 @@ def train_rf(
             for x in test_intervals
         ]
 
+    if test:
+        logger.info("Filtering to chr22 and evaluation intervals for testing...")
```

Comment:
can I ask - why Chr22? Just becuase it's the smallest non-sex Chromosome and it's convention or?

Reply:
I don't really have a good reason other than it's small, and not chr20 since that is being used as the test intervals